code/modules/surgery/organs/heart.dm

```diff
@@ -151,13 +151,14 @@
 
 /obj/item/organ/heart/cybernetic
 	name = "cybernetic heart"
-	desc = "An electronic device designed to mimic the functions of an organic human heart. Also holds an emergency dose of epinephrine, used automatically after facing severe trauma."
+	desc = "An electronic device designed to mimic the functions of an organic human heart. Also holds an emergency dose of epinephrine, used automatically after facing severe trauma. The epinephrine will slowly regenerate after use."
 	icon_state = "heart-c"
 	synthetic = TRUE
 	var/dose_available = TRUE
 
-	var/rid = /datum/reagent/medicine/epinephrine
-	var/ramount = 10
+	var/reagid[] = list(/datum/reagent/medicine/epinephrine, /datum/reagent/medicine/omnizine, /datum/reagent/medicine/atropine, /datum/reagent/medicine/tricordrazine, /datum/reagent/medicine/salbutamol)
+	var/reagamount[] = list(15, 5, 10, 10, 15)
```

Comment:
Our style is to use `var/list/varname` rather than `var/varname[]`.

You could avoid having two lists if you used an associative list: `list(/datum/reagent/medicine/epinephrine = 15, /datum/reagent/medicine/omnizine = 5)` and so on